TESTING_GUIDE.md

```diff
@@ -29,6 +32,14 @@ We have devised our own test framework to test the spec against an OpenSearch cl
 
 ### Running Spec Tests Locally
 
+#### Prerequisites 
+
+Download and install the latest version of Node.js and npm from [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) and run 'npm install'.
```

Comment:
Use backtick quotes so that `npm install` renders.

```suggestion
Download and install the latest version of Node.js and npm from [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) and run `npm install`.
```